src/game/CowGame.tsx

```diff
@@ -28,13 +29,23 @@ export function CowGame() {
 
   // Hide box when pressing Space
   const hideMessageBox = () => setShowMessageBox(false);
-  useKeyPress(["Space", "ArrowUp"], hideMessageBox);
+
+  // Desktop/non-touch device: Key binding to hide message box
+  !!IS_MOBILE && useKeyPress(["Space", "ArrowUp"], hideMessageBox);
+
+  // Mobile/touch device: On touch start hide message box
+  const messageBox = document.getElementById("main-cowgame");
+  if (IS_MOBILE && messageBox) {
+    messageBox.addEventListener("touchstart", hideMessageBox, false);
```

Comment:
Wonder if it's strictly needed to cleanup and 'removeEventListener' as the last part of the triggered fn execution. Probably low overhead.